src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -42,9 +44,6 @@
     <ProjectReference Include="..\..\System.ObjectModel\src\redist\System.ObjectModel.depproj">
       <TargetGroup>netcore50</TargetGroup>
     </ProjectReference>
-    <ProjectReference Include="..\..\System.Runtime\src\redist\System.Runtime.depproj">
```

Comment:
I have bumped the default build to netstandard1.7 and fixed System.ObjectModel to be consistent.

I am trying not to change anything about netcore50aot build in this change. There are number of other projects that have netcore50aot build that have risk of not mixing well with shipped .NET Native. @SedarG is cleaning it up one assembly at a time.